__tests__/pet.test.js

```diff
@@ -0,0 +1,194 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+});
+
+describe('Pet has a name property', () => {
+    it('returns the name of the specified pet', () => {
+        const pet = new Pet('Fido');
+
+        expect(pet.name).toBe('Fido');
+    });
+});
+
+describe('growUp', () => {
+    it('Has an initial age of 0', () => {
+        const pet = new Pet('Fido');
+
+        expect(pet.age).toBe(0);
+    })
+
+    it('causes the pet to age by 1', () => {
+        const pet = new Pet('Fido');
+        pet.growUp();
+        pet.growUp();
+
+        expect(pet.age).toBe(2);
+    });
+
+    it('Throws an error if pet is dead', () => {
+        const pet = new Pet('Fido');
+        pet.age = 30;
+
+        expect(() => pet.growUp()).toThrow(
+            'Sorry, your pet has gone to live on the big farm in the sky'
+            );
+    });
+});
+
+describe('hunger', () => {
+    it('Has an initial hunger value of 0', () => {
+        const pet = new Pet('Fido');
+        expect(pet.hunger).toBe(0);
```

Comment:
I'd add a space between the setup and the expect statement to improve readability